docs/index.md

```diff
@@ -52,6 +52,12 @@ import ReactModal from 'react-modal';
   /* Function that will be run when the modal is requested
      to be closed (either by clicking on overlay or pressing ESC).
      Note: It is not called if isOpen is changed by other means. */}
+     
+  onOverlayRightClick={
+    handleRequestCloseFunc
+    /* Function that will be run when the modal is requested
+     to be closed (either by right clicking on overlay or pressing ESC).
+     Note: It is not called if isOpen is changed by other means. */}
```

Comment:
This comment is out of date given that 06c1c0451230f08966b63fc174160ff8a20cf4f2 changed the prop to be a pass-through to `onContextMenu` instead of actually closing the modal. It is also not triggered by the `esc` key.

I would suggest a straightforward comment like:

> Function that will be run when the context menu is triggered on the overlay (e.g., by right clicking or by pressing the context menu).